gosyntect.go

```diff
@@ -90,6 +98,8 @@ type Client struct {
 	syntectServer string
 }
 
+var client = &http.Client{Transport: &nethttp.Transport{}}
```

Comment:
Using a shared client to avoid opening a new connection every request.

Reply:
Nice